### PR TITLE
BACKLOG-11781: add documentation of each packages

### PR DESCRIPTION
**Pull request by FBerthelot**
## JIRA

https://jira.jahia.org/browse/BACKLOG-11781

## Tests

The following are included in this PR
- [ ] Unit Tests (Most changes _should_ have unit tests)
- [ ] Integration Tests

## Checklist

I have considered the following implications of my change: 

- [ ] Security (in particular for changes to authentication, authorization, data fetching, ...)
- [ ] Performance
- [ ] Migration
- [ ] Code maintainability

## Documentation

- [x] Inline documentation
- [x] Internal Documentation (wiki)
- [x] User-facing Documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,100 +1,118 @@
-# Javascript components
-The goal of this module is to provide multiple javascript [packages](https://github.com/Jahia/javascript-components/tree/master/packages) that can be reused in different javascript applications.
+<h1 align="center">Welcome to Jahia JavaScript Monorepo 👋</h1>
+<p>
+  <img alt="Version" src="https://img.shields.io/badge/version-1.0.0-blue.svg?cacheSeconds=2592000" />
+  <a href="toto.com" target="_blank">
```
Comment (tdraier) [inline README.md]: what am i doing here ?

File: packages/react-router/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## Install
+
+```sh
+yarn add @jahia/react-router
+```
+
+## Usage
+
+TODO, if someone know how to use it ?! :)
```
Comment (tdraier) [inline packages/react-router/README.md]: I'm sure I had an example somewhere, but cannot find it anymore 😬

File: packages/scripts/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+> Jahia internal build script
+
+## Install
+
+```sh
+yarn add @jahia/scripts
+```
+
+## Usage
+
```
Comment (tdraier) [inline packages/scripts/README.md]:
```suggestion
Once installed the following scripts are available and can be used in package.json : 

- jahia-babel-js : generates transpiled JS files in build/js folder, used for "main" entry
- jahia-babel-module : generates transpiled JS files, keeping module import/exports, in build/esm folder, used for "module" entry
- jahia-prepare-build : copy d.ts and resource files to builds directories
- jahia-build : calls jahia-prepare-build , jahia-babel-js and jahia-babel-module
- jahia-publish : publish the module ( deprecated )
- jahia-react2dts  : just call react2dts to generate typings from react component
- jahia-build-umd : build the umd file based on rollup ( deprecated )
```

File: packages/scripts/README.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+```sh
+TODO, if someone know how to use it ?! :)
```
Comment (tdraier) [inline packages/scripts/README.md]:
```suggestion
yarn global add @Jahia/scripts
jahia-build
```

**Comment by FBerthelot** [comment]
@tdraier I have no right to merge it. If you approuved, then you can merge it.

Thanks :-)

**Pull request merged**